routing/router.go

```diff
@@ -1959,16 +1959,13 @@ func (r *ChannelRouter) SendToRoute(hash lntypes.Hash, rt *route.Route) (
 	// mark the payment failed with the control tower immediately. Process
 	// the error to check if it maps into a terminal error code, if not use
 	// a generic NO_ROUTE error.
-	reason := r.processSendError(
-		attempt.AttemptID, &attempt.Route, shardError,
-	)
-	if reason == nil {
-		r := channeldb.FailureReasonNoRoute
-		reason = &r
+	if err := sh.handleSendError(attempt, shardError); err != nil {
```

Comment:
Nice, yeah I think we just forgot to update this call site to use the new method after it was added.